From c8977f6d268462e237cee38be6159c7bb701c62e Mon Sep 17 00:00:00 2001
From: Anton Khirnov <anton@khirnov.net>
Date: Wed, 31 Oct 2012 21:15:41 +0100
Subject: [PATCH 167/278] avconv: do not free muxed packet on streamcopy.

The packet belongs to lavf, the caller must not free it or access it in
any other way.
No change in practice, since destruct is set to NULL anyway.
---
 avconv.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/avconv.c b/avconv.c
index d0902a6..964847a 100644
--- a/avconv.c
+++ b/avconv.c
@@ -1045,7 +1045,6 @@ static void do_streamcopy(InputStream *ist, OutputStream *ost, const AVPacket *p
 
     write_frame(of->ctx, &opkt, ost);
     ost->st->codec->frame_number++;
-    av_free_packet(&opkt);
 }
 
 static void rate_emu_sleep(InputStream *ist)
-- 
1.7.9.5

